packages/dask-task-models-library/src/dask_task_models_library/container_tasks/events.py

```diff
@@ -76,6 +77,7 @@ class Config(BaseTaskEvent.Config):
 
 class TaskLogEvent(BaseTaskEvent):
     log: str
+    log_level: int = logging.INFO
```

Comment:
MINOR:

SEE https://github.com/python/cpython/blob/main/Lib/logging/__init__.py#L98-L124
```python
# logging/__init__.py

CRITICAL = 50
FATAL = CRITICAL
ERROR = 40
WARNING = 30
WARN = WARNING
INFO = 20
DEBUG = 10
NOTSET = 0
```

```cmd
>> import logging
>> [logging.NOTSET, logging.DEBUG, logging.INFO, logging.ERROR, logging.CRITICAL]
[0, 10, 20, 40, 50]
```

therefore

```python
LogLevelInt = Literal[0,10,20,30,40,50] # or alternatively create a nice ``conint``

log_level: LoglLevelInt = logging.INFO
```

It would be also nice to have somehow this functionality later used in https://github.com/ITISFoundation/osparc-simcore/blob/71e4bbc6e5fff43c7e3ca61a74ee79cd7d02c0bb/packages/settings-library/src/settings_library/utils_logging.py

Reply:
Thinking of how the fronted will interpret this data. Wondering if we should maybe transform log level to a string.

Reply:
I do agree with your idea, but I find convoluted that I have to use the "real" values instead of the constants. I do not really see a way around. so I think I will keep it simple, unless you have a better idea.